### PR TITLE
BaSP-MR-79: List a table of trainers and delete petition functionality 

### PR DESCRIPTION
**Pull request by maurojjzz**
### Description

My [Trello's ticket](https://trello.com/c/WnEW2EAr)  describes that i had to list the trainer that are created on a given database within a table i which i also had to add extras buttons: 'Edit' and  'Delete'  but only the delete button would have functionality with it petition.

I used the global Modals on my table and made a little change on it (with consent of the creator). 

### Changes Made
On Trainers component
- index.js and its css added.
- created Table component that contains an index.js .
- created a table.module.css which i used to give styles to the table.
- added buttons on table and gave them styles as well.

### Checklist

- [x] Show table with list: Columns: full name (first name + last name), email, phone, city, is active, salary
- [x]  Add delete row functionality
- [x]  Add actions icons (delete, edit) for each row
- [x] Warning message for delete
- [x] Success message for confirm

### CODE REVIEW

**Comment by maurojjzz** [comment]
[TRAINER - List/delete table](https://trello.com/c/WnEW2EAr/79-trainer-list-delete-table)

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+  return (
+    <form className={styles.form}>
+      <div className={styles.subContainer}>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}> First Name</label>
+          <input
+            className={styles.input}
+            //firstName="firstName"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:43]: remove this comment

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            type="text"
+            value={trainer.firstName}
+            onChange={onChargeInput}
+          />
+        </div>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}> Last Name</label>
+          <input
+            className={styles.input}
+            //lastName="lastName"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:53]: remove this comment

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            type="text"
+            value={trainer.lastName}
+            onChange={onChargeInput}
+          />
+        </div>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}>DNI</label>
+          <input
+            className={styles.input}
+            //dni="dni"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:63]: remove this comment

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            type="number"
+            value={trainer.dni}
+            onChange={onChargeInput}
+          />
+        </div>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}>Phone</label>
+          <input
+            className={styles.input}
+            //phone="phone"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:73]: remove this comment

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            type="number"
+            value={trainer.phone}
+            onChange={onChargeInput}
+          />
+        </div>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}>Email</label>
+          <input
+            className={styles.input}
+            //email="email"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:83]: remove this comment

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            type="text"
+            value={trainer.email}
+            onChange={onChargeInput}
+          />
+        </div>
+        <div className={styles.inputContainer}>
+          <label className={styles.label}>City</label>
+          <input
+            className={styles.input}
+            //city="city"
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:93]: remove this comment

File: src/Components/Trainers/TableTrainers/index.jsx [hunk 7]
```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import styles from './table.module.css';
+import ModalConfirm from '../../Modals/ModalConfirm/index';
+import ModalSuccess from '../../Modals/ModalSuccess/index';
+// import ToastError from '../Modals/ToastError/index';
```
Comment (NicoCagnina) [inline src/Components/Trainers/TableTrainers/index.jsx:5]: remove this comment

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

**Review by FrancoLelli — APPROVED** [review]
Nice Job Mauro!

**Review by CristianLotorto — APPROVED** [review]
great job!

File: src/Components/Trainers/FormTrainers/index.jsx [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    firstName: '',
+    lastName: '',
+    dni: '',
+    phone: '',
+    email: '',
+    city: '',
+    salary: ''
+  });
+
+  const onChargeInput = (e) => {
```
Comment (NicoCagnina) [inline src/Components/Trainers/FormTrainers/index.jsx:15]: should be change not charge

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

**Review by NicoCagnina — APPROVED** [review]

**Pull request merged**